source/faucet/main.d

```diff
@@ -347,7 +347,9 @@ public class Faucet : FaucetAPI
 
         if (this.state.utxos.storage.length > config.tx_generator.merge_threshold)
         {
-            auto tx = this.mergeTx(this.state.utxos.byKeyValue().take(uniform(10, 100, rndGen)));
+            auto tx = this.mergeTx(this.state.owned_utxos.byKeyValue
+                .map!(kv => tuple(kv.value.output, kv.key))
+                .take(uniform(1, this.state.owned_utxos.length, rndGen)));
```

Comment:
Faucet might have a very large amount of owned UTXO, so that number might be quite high, and it will be rejected based on length.

Reply:
In that case I'll put back the 100 limit.

Reply:
I also reverted the switch to using `owned_utxos` which was not relevant to this commit.